### PR TITLE
Add statemine

### PR DESCRIPTION
**Pull request by alistair-singh**
Merged in `bko-transfer-asset-via-bridge` statemine code.

Snowfork companion: https://github.com/Snowfork/snowbridge/pull/829

Resolves: [SNO-433](https://linear.app/snowfork/issue/SNO-433)

### CODE REVIEW

File: pallets/parachain-system/src/lib.rs [hunk 1]
```diff
@@ -1059,8 +1059,8 @@ impl<T: Config> Pallet<T> {
 					max_capacity: 10,
 					max_total_size: 10_000_000_u32,
 					max_message_size: 10_000_000_u32,
-					msg_count: 10,
-					total_size: 10_000_000_u32,
+					msg_count: 5,
+					total_size: 5_000_000_u32,
```
Comment (doubledup) [inline pallets/parachain-system/src/lib.rs]: Why was this doubled in the merge?

**Review by doubledup — APPROVED** [review]

**Review by vgeddes — APPROVED** [review]
+1

File: pallets/parachain-system/src/lib.rs (hunk 1 above)
Comment (alistair-singh) [reply pallets/parachain-system/src/lib.rs]: I made a mistake during the merge and chose the wrong conflict resolution. I have reverted this though in [94c880c](https://github.com/Snowfork/cumulus/pull/24/commits/94c880ca571aa49db740d0f47906c906952d66c7).

This is master branch which is more recent and was updated during the wococo-rococo bridge work:
https://github.com/paritytech/cumulus/blame/master/pallets/parachain-system/src/lib.rs#L1042-L1065

In the `bko-transfer-assets-via-bridge` its two months old:
https://github.com/paritytech/cumulus/blame/bko-transfer-asset-via-bridge/pallets/parachain-system/src/lib.rs#L1045-L1068

**Review by yrong — APPROVED** [review]
LGTM!

**Pull request merged**